scripts/metadata_table.py

```diff
@@ -64,6 +64,7 @@
   relative_path = <relative path>
   dependencies = <dependencies>
   module = <module name> # only needed if module name differs from filename
+  source_path = <relative source directory of Fortran source (if different)>
   dynamic_constituent_routine = <routine name>
```

Comment:
oops! I know this is unrelated to your changes, but can you remove this residual dynamic constituents line left by me?